web/app/themes/radcliffe/single-news.php

```diff
@@ -71,7 +71,6 @@
 
                 <ul class="single-news__links section-inner thin">
                     <li><a href="/news">More news</a></li>
-                    <li>|</li>
```

Comment:
Why was the separator removed?
I realise that is is consuming a &lt;li&gt; element and is not symanic however, the distinction of separation between the links cannot be achieved without a replacement.

Reply:
Ok. Reverted to original.

Reply:
I think the move to remove the &lt;li&gt; may have been related to us reaching AA standard in accessibility - can you advise please Stephanie? Thank you

Reply:
"A" in this case, which must be met before meeting "AA". Have a look at:
•	Understanding Success Criterion 1.3.1: Info and Relationships - https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html
•	Using ol, ul and dl for lists or groups of links - https://www.w3.org/WAI/WCAG21/Techniques/html/H48
•	Using Lists Correctly - https://webaim.org/techniques/semanticstructure/#lists

WebAIM Semantic Structure – Using Lists Correctly
HTML lists - `ul, ol, dl` - also convey a hierarchical content structure. Each of these has rules regarding their use as well. Unordered lists should be used when there is no order of sequence or importance. Ordered lists suggest a progression or sequence. Definition lists should be used explicitly for presenting a structure for definitions. **As with heading, lists should be used correctly and for the right purposes.** Unordered and ordered lists should always contain list items. Definition lists must always have definition descriptions. Empty lists are incorrect HTML. **Lists should never be used for merely indenting or other layout purposes.** Nested lists should be coded properly.